modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx

```diff
@@ -12,8 +12,50 @@
  * details.
  */
 
+import ClayPopover from '@clayui/popover';
 import classNames from 'classnames';
-import React from 'react';
+import React, {useState} from 'react';
+
+export function NotificationQueueEntryToDataRenderer({value}: {value: string}) {
+	const [isPopoverVisible, setPopoverVisible] = useState(false);
+
+	const POPOVER_MAX_WIDTH = 256;
+
+	const to = value;
+	const toArray = to.split(',');
+	const lastItem = toArray.length > 1 ? toArray.pop() : null;
+	const toTooltipText =
+		toArray.length > 1
+			? toArray.join(',') +
+			  ' ' +
+			  Liferay.Language.get('and').toLowerCase() +
+			  lastItem +
+			  '.'
+			: to;
```

Comment:
Hey @alinedoleron, to make the code inside the component as clean as possible, we can create a function outside the component to do this, and on line `57` we just call the function passing the `value` as a parameter, like:

```tsx

const getToTooltipText = (value: string) => {
	const toArray = value.split(',');
	const lastItem = toArray.length > 1 ? toArray.pop() : null;

	return toArray.length > 1
		? toArray.join(',') +
				' ' +
				Liferay.Language.get('and').toLowerCase() +
				lastItem +
				'.'
		: value;
};
```